src/screens/map-view-screen.tsx

```diff
@@ -51,9 +54,12 @@ export const MapViewScreen = () => {
             let locationz = await Location.getCurrentPositionAsync({});
```

Comment:
Clean up this async function inside of the useEffect hook. I am trying to remember if you can use `useEffect(async () => ())`. If not, then I would define a separate function such as
```ts
const getLocation = async () => { ... }
useEffect(() => getLocation());
```